client/reader/recent/index.tsx

```diff
@@ -147,50 +149,88 @@ const Recent = () => {
 		}
 	}, [ isWide, data?.items, selectedItem ] );
 
+	const [ hasLoadedItems, setHasLoadedItems ] = useState( false );
+
+	useEffect( () => {
+		const isEmpty = ! data?.items || data.items.length === 0;
```

Comment:
Could we move this outside and "above" of the `useEffect` callback and then remove the duplicate statement on line 166?

Reply:
That makes sense to me! I updated the code in 33cce9e04576b32b8a9493b44834643855ac3709